doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+# Practical Overview of Explainability
+
+While the applications of machine learning are impressive many models provide predictions that are hard to interpret or reason about. This limits their use in many cases as often we want to know why and not just what a model made a prediction. Alarming predictions are rarely taken at face value and typically warrant further analysis. Indeed, in some
+cases explaining the choices that a model makes could even become a potential [legal requirement](https://arxiv.org/pdf/1711.00399.pdf). The following is a non-rigorous and practical overview of explainability and the methods alibi provide.
+
+**Explainability provides us with algorithms that give insights into trained models predictions.** It allows
+us to answer questions such as:
+- How does a prediction change dependent on feature inputs?
+- What features are Important for a given prediction to hold?
+- What features are not important for a given prediction to hold?
+- What set of features would you have to minimally change to obtain a new prediction of your choosing?
+- How does each feature contribute to a model's prediction?
+
+The set of insights available are dependent on the trained model. For instance, If the model is a regression it makes sense to ask how the prediction varies for some feature. Whereas, it doesn't make sense to ask what minimal change is required to obtain a new classification. Insights are constrained by:
```

Comment:
If --> if

Reply:
Kind of like the example, would just clarify slightly e.g.:

"regression" --> "regressor"
"new classification" --> "new class prediction"